llpc/patch/llpcPatchDescriptorLoad.cpp

```diff
@@ -41,17 +41,10 @@
 using namespace llvm;
 using namespace Llpc;
 
-namespace llvm
-{
-
-namespace cl
-{
-
-extern opt<bool> EnableShadowDescriptorTable;
-
-} // cl
-
-} // llvm
+// -enable-shadow-desc: enable shadow desriptor table
+static cl::opt<bool> EnableShadowDescriptorTable("enable-shadow-desc",
```

Comment:
EnableShadowDescriptorTable -> EnableShadowDescTable
enable-shadow-desc -> enable-shadow-desc-table
Just to keep all namings consistent with ShadowDescTablePtrHigh

Reply:
Can't change the option name as it is used in compiler_solution_llpc.cpp.

Reply:
OK. Let's keep it as-is